README.md

```diff
@@ -270,8 +270,9 @@ Calling this method after the response has ended/closed is a NOOP.
 
 Unless you specify a `Content-Length` header yourself, HTTP/1.1 responses
 will automatically use chunked transfer encoding and send the respective header
-(`Transfer-Encoding: chunked`) automatically. If you know the length of your
-body, you MAY specify it like this instead:
+(`Transfer-Encoding: chunked`) automatically. If `Transfer-Encoding` is already given
+the `Content-Length` will be removed.
```

Comment:
Either I'm missing something obvious here or the current implementation does not match with this documentation?!

Should the consumer really have any control over enforcing any `Transfer-Encoding` at all? My vote would be no, because this should be transparently handled by the server.